draft-ietf-ppm-dap.md

```diff
@@ -825,10 +825,10 @@ opaque TaskID[32];
 The task ID value MUST be a globally unique sequence of bytes. Each task has
 the following parameters associated with it:
 
-* `leader_aggregator_endpoint`: A URL relative to which the Leader's API
-  endpoints can be found.
-* `helper_aggregator_endpoint`: A URL relative to which the Helper's API
-  endpoints can be found.
+* `leader_aggregator_server`: A URL relative to which the Leader's API resources
```

Comment:
I am not sure `server` is appropriate here (or for `helper_aggregator_server` below) -- I think either `{leader,helper}_aggregator_endpoint` or perhaps `{leader,helper}_aggregator_url` would be clearer. I'd expect a server to indicate a hostname, while this is a full URL including a protocol & path.

Reply:
(I'd go with "endpoint", but I think that an endpoint might also indicate a particular HTTP verb; if that's correct, I'd go with "URL.")

Reply:
`_url` seems better since that's what the text on the right of the colon says. I want to avoid "endpoint" as the httpdir review flagged that word.